modules/thermal_hydraulics/include/physics/THMWCNSFVFlowPhysics.h

```diff
@@ -0,0 +1,69 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ThermalHydraulicsFlowPhysics.h"
+#include "WCNSFVFlowPhysics.h"
+
+class PhysicsFlowBoundary;
+
+/**
+ * Sets up the single-phase flow equations using Euler's equations and a RDG discretization
+ */
+class THMWCNSFVFlowPhysics : public virtual ThermalHydraulicsFlowPhysics, public virtual WCNSFVFlowPhysics
```

Comment:
This will need a test of course

Reply:
oh this is like super not ready. will do

Reply:
Yeah I realized that later in the review 😄